### DISCUSSION TITLE
SSH into pods

### DISCUSSION THREAD

**Discussion by regressEdo** [MuhammedKalkan/OpenLens#138] [Q&A]
Hello, 

I hope this argument haven't been treated before to not bring unneeded noise,
I was a Lens user before the end of 2022 and after switching to OpenLens I noticed something missing, I can't find the button to be able to ssh into a pod. 
If it's just me not seeing it - please just point me to it - or in case it's not present I think it's something really nice to have.

Thank you so much for your time and effort!

**Comment by rdeepc** [comment]
Hi @ebeltramo96 ,

Its not there in the available builds in this repo, If you want to those features,

You can use extensions,  https://github.com/alebcay/openlens-node-pod-menu

For details: #83 

Thanks

**Reply by regressEdo** [reply]
Thanks!!

**Comment by regressEdo** [comment]
closed
